app/utils/sentry.ts

```diff
@@ -39,9 +40,20 @@ export function initializeSentry() {
         return;
     }
 
+    const isBetaApp = DeviceInfo.getBundleId && DeviceInfo.getBundleId().includes('rnbeta');
+    const mmConfig = {
+        environment: isBetaApp ? 'beta' : 'production',
+        tracesSampleRate: isBetaApp ? 1.0 : 0.2,
+        sampleRate: isBetaApp ? 1.0 : 0.2,
+        attachStacktrace: isBetaApp, // For Beta, stack traces are automatically attached to all messages logged
+        logLevel: isBetaApp ? Sentry.LogLevel.Debug : Sentry.LogLevel.Error,
```

Comment:
I still think you need soke logic in captureJsException, right?

Reply:
added